package.json

```diff
@@ -48,7 +68,24 @@
     "@types/mocha": "^9.1.1",
     "@types/node": "^17.0.42",
     "mocha": "^10.0.0",
+    "packemon": "^2.4.0",
     "ts-node": "^10.8.1",
     "typescript": "^4.7.3"
-  }
+  },
+  "packemon": [
+    {
+      "inputs": {
+        "index": "src/check-ends-with-period.ts"
+      },
+      "format": "esm",
+      "platform": "browser"
+    },
+    {
+      "inputs": {
+        "index": "src/check-ends-with-period.ts"
+      },
+      "format": "cjs",
+      "platform": "node"
+    }
```

Comment:
This config generate `index-wrapper.mjs`.
It will avoid dual pacakge hazard.
https://github.com/milesj/packemon/issues/133

Previous version force me to use `type: commonjs`.
It blocked to use ts-node + mocha + esm testing.
But, this issue has been resolved.